Source/WebCore/html/canvas/OESVertexArrayObject.h

```diff
@@ -25,30 +25,24 @@
 
 #pragma once
 
-#if ENABLE(WEBGL)
```

Comment:
Why are the include guards for ENABLE(WEBGL) being removed in this PR? I think they need to remain in order to accomodate WebKit ports that don't compile in WebGL support.

Reply:
- Guards inside extension headers are redundant because these header files are included only by `WebGL{2}RenderingContext*` units that have their own guards in place.
- Many other WebGL extension headers do not have these guards, suggesting that they are here only for historical reasons.

Reply:
OK. Please explicitly mention in the commit message the removal of redundant ENABLE(WEBGL) guards from various WebGL extension headers, since those are already guarded elsewhere.

Reply:
Done.